libraries/helpers.rb

```diff
@@ -69,13 +79,17 @@ def jks_opts(prefix)
     end
 
     # Return hash with SSL options for OpenSSL
-    def ssl_opts
+    def cdap_ssl_opts(prefix = 'dashboard')
       ssl = {}
-      ssl['keypath'] = node['cdap']['cdap_security']['dashboard.ssl.key']
-      ssl['certpath'] = node['cdap']['cdap_security']['dashboard.ssl.cert']
+      ssl['keypath'] = node['cdap']['cdap_security']["#{prefix}.ssl.key"]
```

Comment:
original comment applied to when ``cdap_property`` returned the value. no longer relevant